feat(scim): okta update user attribute docs (#6447)
getsentry/sentry-docs: src/docs/product/accounts/sso/okta-sso/okta-scim.mdx

![Okta Role Restricted Role Select](./okta-role-restricted-sentry-role-select.png)

> ### Updating user attributes

Reviewer: ```suggestion
### Updating User Attributes
```

> 
> Sentry currently only supports updating user attributes for organization-level roles. Once you have added the `sentryOrgRole` attribute to your Okta application profile, you can edit the attribute for users assigned to your application.
> 
> ![Okta Edit Attribute](okta-edit-sentryOrgRole-attribute.png)
> 
> Editing a user's role attribute to blank will remove the organization-level role in Sentry. The user's role in Sentry will default to the organization role selected in your Auth settings.
> 

Reviewer: ```suggestion
Currently, Sentry only supports the ability to update user attributes for organization-level roles. You'll be able to edit the attribute for users assigned to your application once you've added the `sentryOrgRole` attribute to your Okta application profile.

![Okta Edit Attribute](okta-edit-sentryOrgRole-attribute.png)

The user's role in Sentry will reflect their organization role in your Auth settings. This means that if you change a user's attribute to blank, their organization-level role will be removed from Sentry. 

```

### How do I remove the organization-level role for a user that I set via Okta?

> Edit the user's role attribute to blank.

Reviewer: ```suggestion
Change the user's role attribute to blank.
```
